### PR TITLE
Document `@bp` macro

### PR DESCRIPTION
**Pull request by antoine-levitt**
I was about to request a functionality similar to the `keyboard` function in Matlab. Turns out it's already there, so here's a doc PR instead.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,6 +159,40 @@ Stacktrace:
 [...]
 ```
 
+### Breakpoint at specific location
+It is sometimes useful to programmatically decide when to break and drop to the debugger to inspect your surroundings; this is for instance done in Matlab/Octave with the `keyboard` command. You can use the `@bp` macro in JuliaInterpreter to do this:
```
Comment (KristofferC) [inline README.md]: `@bp` is reexported so no need to mention (or import) JuliaInterpreter

Comment (KristofferC) [reply README.md]: Also, note that you can have conditional breakpoints. So `bp add f:2 x < 0` or something to break on line 2 if `x < 0`. But perhaps you already know that.

**Comment by antoine-levitt** [comment]
Even better!

File: README.md [hunk 2]
```diff
@@ -159,6 +159,40 @@ Stacktrace:
...
+```
```
Comment (pfitzseb) [inline README.md]: Can we make this a doctest?

File: README.md (hunk 1 above)
Comment (antoine-levitt) [reply README.md]: Yeah I know, but this `keyboard` thing is really convenient for more complicated codepaths; plus I often find it easier to do things in the source code than in the REPL. I'll mention that this is an alternative.

File: README.md (hunk 2 above)
Comment (fredrikekre) [reply README.md]: Probably not -- it requires user input in the debug mode.

File: README.md (hunk 1 above)
Comment (antoine-levitt) [reply README.md]: I changed to make clear that the main motivation is to put the breakpoint in the source, not really conditional breakpoints. Also added a ref to R which has a similar thing.

File: README.md (hunk 2 above)
Comment (pfitzseb) [reply README.md]: Ah, good point.

**Pull request merged**